### PR TITLE
Updated development stack and fixed CI.

### PR DESCRIPTION
**Pull request by alexdesignworks**
## Please note: This PR description applies to all PRs for other Tide modules and Tide profile.

# Changes
- Removed all non-module files
- Updated configuration to use Dev Tools (https://github.com/dpc-sdp/dev-tools)
- Fixed tests

# Changes in all repos
- Wiki and Projects were disabled
- Only `Squash merging` is allowed to keep history clean (we are running a public project!)

See https://github.com/dpc-sdp/dev-tools for more information about changes to how the development process changes.

**Please make sure that you see file changes section with inline comments for 'questionable' changes**

# How to test
1. Checkout this branch
2. Run `. dev-init.sh`
3. Make sure that you have `jq` installed locally (required for module development; see https://github.com/dpc-sdp/dev-tools#faqs for explanation why)
4. `pygmy up`
5. `ahoy build`

Please note that CI now runs exactly the same steps as a developer would run. Consistency FTW!

# How to merge
Due to the fact that all modules need to have new workflow and Dev Tools supported and have these changes in `ci` branches in each project, this branch need an additional commit to change the `dev-ci` constraint in `composer.json` to a proper version.
In other words, this PR should be merged once all PRs were reviewed and approved.

I will be merging all these PRs once they are ready.

# Further work
Once all PRs are merged, it will allow to make any further changes across modules (including Drupal core update) significantly easier.

### CODE REVIEW

File: tide.install [hunk 1]
```diff
@@ -43,10 +43,13 @@ function tide_install() {
...
-  // be bootstrapped. We are already using installation phase to exclude redis
-  // configuration in settings.php and to install the module itself.
-  if (drupal_get_path('module', 'bay') && drupal_get_path('module', 'redis')) {
-    \Drupal::service('module_installer')->install(['redis']);
+  $moduleHandler = \Drupal::service('module_handler');
+  if ($moduleHandler->moduleExists('bay')) {
+    // Workaround for redis module: it must be enabled before configuration can
+    // be bootstrapped. We are already using installation phase to exclude redis
+    // configuration in settings.php and to install the module itself.
+    if (drupal_get_path('module', 'bay') && drupal_get_path('module', 'redis')) {
```
Comment (kurtfoster) [inline tide.install]: @alexdesignworks , we need to remove `drupal_get_path('module', 'bay')`. The module is empty.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,44 +1,26 @@
...
-      - run:
-          name: Install dev dependencies
-          command: make install
+      - run: .circleci/build.sh
+      - run: .circleci/test.sh
       - run:
-          name: Lint code
-          command: make lint
+          name: Copy artifacts
+          command: .circleci/test-artifacts.sh
```
Comment (kurtfoster) [inline .circleci/config.yml]: @alexdesignworks , where is this file?
